optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/statistic/PureSingleStatistic.java

```diff
@@ -30,7 +30,7 @@
 import org.optaplanner.benchmark.impl.statistic.single.pickedmovetypestepscore.PickedMoveTypeStepScoreDiffSingleStatistic;
 
 /**
- * 1 statistic of {@link SingleBenchmarkResult}
+ * 1 statistic of {@link SingleBenchmarkResult}.
```

Comment:
I'd argue (from a language syntactical standpoint) that there actually is a verb implicitly in the sentence (therefore, it is a sentence):
```
(This is) 1 statistic of (a) {@link SingleBenchmarkResult}.
```
I may be wrong on that account thought. Anyway, the `JavadocStyle` check expects the first sentence to be a summary of what the method/class/field/etc does/is, and it makes sense for it to expect some sort of punctuation at the end. This can be configured (either disabling the first sentence check or supplying your own regex) and I will do it if you want to, but I'm in favor of just adding the dot here.